src/main/java/gift/entity/User.java

```diff
@@ -0,0 +1,31 @@
+package gift.entity;
+
+public class User {
+
+    private Long id;
+    private String email;
+    private String password;
+
+    public User() {}
+
+    public User(String email, String password) {
+        this.email = email;
+        this.password = password;
+    }
```

Comment:
```suggestion
public class User {

    private final Long id;
    private final String email;
    private final String password;

    protected User() {}

    public User(String email, String password) {
        this.email = email;
        this.password = password;
    }
```

- 나중에 JPA를 사용할 것을 위해 기본 생성자를 둔 걸까요? 그런게 아니라면 사용하지 않는 코드는 제거해주세요. (필요할 때 생성해주셔도..)
- 객체를 불변(Immutable)하게 설계해보는건 어떨까요? [불변객체](https://www.brainbackdoor.com/95248237-c6a4-4489-a51e-6c548c0c103c)에 대해서도 학습해보시겠어요?

Reply:
db에서 생성하는 id를 제외한 필드들을 final로 선언하였습니다.